mal_scrapper.py

```diff
@@ -0,0 +1,209 @@
+#!/usr.bin/env python3
+##
+# Dakara Project
+#
+
+import bs4 as bs
+import json
+import html
+import os
+import re
+import requests
+
+json_file_path = './mal_scrapper.json'
+
+
+class File():
```

Comment:
`open` by itself is a context manager, so you don't need this bypass class. You simply do:

```python
with open(filename, "w") as file:
    content = file.read()
```